src/odemis/acq/align/test/keypoint_test.py

```diff
@@ -78,8 +78,10 @@ def test_image_pair(self):
         sem_img = preprocess(sem_img, image_pair[1][1], image_pair[1][2],
                              image_pair[1][3], image_pair[1][4], True)
         # execute the algorithm to find the transform between the images
-        tmat, _, _, _, _ = keypoint.FindTransform(tem_img, sem_img)
-
+        try:
```

Comment:
We might also use the unittest.expectedfailure here. However, I thought it might be a good idea to only skip the test in case of a ValueError, so if anything else goes wrong, we will be notified.